doc/sphinx/source/inspection/profiling.rst

```diff
@@ -0,0 +1,25 @@
+.. inspection/profiling.rst:
+
+.. _profiling: 
+
+Profiling Performance
+#####################
+
+nGraph currently uses ``nbench`` to assess and debug performance. 
+
+This tool can benchmark any nGraph ``.json`` model with a given backend.
+
+Compile and run with:
+
+.. code-block:: console
+
+   $ g++ ./nbench.cpp
+             -std=c++11
+             -I$HOME/ngraph_dist/include
+             -L$HOME/ngraph_dist/lib
+             -lngraph
+             -o nbench
+   $ env LD_LIBRARY_PATH=$HOME/ngraph_dist/lib env NGRAPH_INTERPRETER_EMIT_TIMING=1 ./nbench
+
+
```

Comment:
I would also add something like this: 
nbench is built along with ngraph. If you are in ngraph/build , then it is src/tools/nbench.
One can run it as `src/tools/nbench/nbench -b CPU - i 1 -f <serialized_json file>`
Here are the current options for nbench
```DESCRIPTION
    Benchmark nGraph JSON model with given backend.
SYNOPSIS
        nbench [-f <filename>] [-b <backend>] [-i <iterations>]
OPTIONS
        -f|--file                 Serialized model file
        -b|--backend              Backend to use (default: CPU)
        -d|--directory            Directory to scan for models. All models are benchmarked.
        -i|--iterations           Iterations (default: 10)
        -s|--statistics           Display op statistics
        -v|--visualize            Visualize a model (WARNING: requires Graphviz installed)
        --timing_detail           Gather detailed timing
        -w|--warmup_iterations    Number of warm-up iterations
        --no_copy_data            Disable copy of input/result data every iteration
        --dot                     Generate Graphviz dot file
        --double_buffer           Double buffer inputs and outputs```

One can also generate serialized files from the framework , say in TF, use NGRAPH_ENABLE_SERIALIZE=1.
And Compiling nbench is controlled by a compile flag `NGRAPH_TOOLS_ENABLE` which is default to true currently.